AceComm-3.0/ChatThrottleLib.lua

```diff
@@ -273,6 +276,9 @@ function ChatThrottleLib.Hook_SendAddonMessage(prefix, text, chattype, destinati
 	self.avail = self.avail - size
 	self.nBypass = self.nBypass + size	-- just a statistic
 end
+function ChatThrottleLib.Hook_SendAddonMessageLogged(prefix, text, chattype, destination, ...)
+	ChatThrottleLib.Hook_SendAddonMessage(prefix, text, chattype, destination, ...)
+end
```

Comment:
Is there a reason for this indirection beyond possibly future changes? Or could the cook above just call Hook_SendAddonMessage?

Reply:
Purely just in case there's a need for future changes. I suspect the risk is extremely low, but up to you - can remove if you'd prefer it gone.

Reply:
Its fine for clarity, I suppose.